mix.exs

```diff
@@ -187,10 +187,7 @@ defmodule AeMdw.MixProject do
       plt_add_apps: [:mix],
       plt_core_path: "priv/plts",
       plt_file: {:no_warn, "priv/plts/dialyzer.plt"},
-      flags: [
-        :unmatched_returns,
-        :race_conditions
-      ]
+      flags: [:unmatched_returns]
```

Comment:
The removed flag was removed from the library itself: https://github.com/jeremyjh/dialyxir/pull/471